test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -30,7 +30,8 @@ MACRO_EXPANSION        = YES
 EXPAND_ONLY_PREDEF     = YES
 SORT_BRIEF_DOCS        = YES
 TOC_INCLUDE_HEADINGS   = 2
-HTML_EXTRA_STYLESHEET = ${SEQAN3_DOXYGEN_SOURCE_DIR}/test/documentation/seqan3.css
+HTML_EXTRA_STYLESHEET  = ${SEQAN3_DOXYGEN_SOURCE_DIR}/test/documentation/seqan3.css
+INPUT_ENCODING         = ISO-8859-1
```

Comment:
😱 why not UTF8? 😱

Reply:
How could I have missed that :see_no_evil:

Reply:
I thought this is UTF8? @eseiler?

Reply:
No, this is Latin-1. It should be "UTF-8" instead.

Reply:
ops